testnets/neutrontestnet/chain.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "neutrontestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Neutron Testnet",
+  "chain_id": "baryon-1",
+  "bech32_prefix": "neutron",
+  "daemon_name": "neutrond",
+  "node_home": "$HOME/.neutrond",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "untrn",
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.05
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/neutron-org/neutron",
+    "recommended_version": "v0.2.0",
+    "compatible_versions": [],
+    "cosmos_sdk_version": "0.45",
+    "tendermint_version": "0.34",
+    "cosmwasm_version": "0.30",
+    "cosmwasm_enabled": true,
+    "ibc_go_version": "4.2.0",
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/cosmos/testnets/master/replicated-security/baryon-1/baryon-1-genesis.json"
+    }
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "e2c07e8e6e808fb36cca0fc580e31216772841df",
+        "address": "p2p.baryon.ntrn.info:26656",
+        "provider": "Neutron"
```

Comment:
Isn't it would be better to write P2P here?

Reply:
Ready